aws/sdk/gradle.properties

```diff
@@ -36,7 +36,8 @@ aws.services.smoketest=\
     +qldbsession,\
     +s3,\
     +sts,\
-    +transcribestreaming
+    +transcribestreaming, \
+    +transcribe
```

Comment:
Was this required to make CI pass, or to make the example compile locally?

Reply:
the example relies on the non-streaming Transcribe API